kube/kd/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
I don't think this needs to be in the repository as it's created once?

Reply:
Removed.